### DISCUSSION TITLE
How to run codes inside a CONDITIONAL block

### DISCUSSION THREAD

**Discussion by The11thPencil** [Gandi-IDE/custom-extension#108] [Q&A] [answered]
![image](https://github.com/Gandi-IDE/custom-extension/assets/124476952/5cba75db-6e6a-4505-a6cf-adf2a48478ea)
![image](https://github.com/Gandi-IDE/custom-extension/assets/124476952/1f3d744b-da15-49d4-a633-bf4bb485e479)
![image](https://github.com/Gandi-IDE/custom-extension/assets/124476952/293f01bc-8e03-4614-a66e-4fe4db6433e2)

I am using the normal remote extension format. I tried asking the Turbowarp community, but their solution only works in Gandi. I can figure it out by reading codes in this repo, but no extension uses conditional or loop blocks. 

The code(just incase): https://github.com/fath11/custom-extension/blob/main/Fath/Fath%20Ghost.js

**Comment by FurryR** [comment, marked as answer]
Hello there! I'm FurryR (Shinogi Kumagai), an unofficial extension developer. I'm here to help you.

# What to do

To run statements (branch) in the same thread, please use [util.startBranch](https://github.com/Gandi-IDE/scratch-vm/blob/develop/src/blocks/scratch3_control.js#L127).

# How to use

```js
fn(_, util) {
  util.startBranch(1, false);
}
```

---

*In order to publish your extension, you need to invite a **translator** to translate your extension into **Simplified Chinese**.*

**Reply by The11thPencil** [reply]
Thank you very much
